controller/volume_rebuilding_controller.go

```diff
@@ -192,12 +279,130 @@ func (vbc *VolumeRebuildingController) reconcile(volName string) (err error) {
 	}()
 
 	rebuildingAttachmentTicketID := longhorn.GetAttachmentTicketID(longhorn.AttacherTypeVolumeRebuildingController, volName)
+	defer func() {
+		if err != nil {
+			delete(va.Spec.AttachmentTickets, rebuildingAttachmentTicketID)
+			return
+		}
+	}()
```

Comment:
deleteVATicketRequired := true
```suggestion
	defer func() {
        if deleteVATicketRequired {
			delete(va.Spec.AttachmentTickets, rebuildingAttachmentTicketID)
		}
	}()
```

Reply:
Modified. Thanks.